HeroHours/views.py

```diff
@@ -109,21 +114,28 @@ def handle_bulk_updates(user_id):
     if user_id == '-404':
         if not os.environ.get('DEBUG', 'False') == 'True':
             return redirect('index')
-        getall = models.Users.objects.filter(Checked_In=False)
+        getall = models.Users.objects.filter(Checked_In=False, Is_Active=True)
     else:
-        getall = models.Users.objects.filter(Checked_In=True)
+        getall = models.Users.objects.filter(Checked_In=True, Is_Active=True)
 
     for user in getall:
-        log = models.ActivityLog(user_id=user.User_ID,entered=user.User_ID, operation='Check In' if user_id == '-404' else 'Check Out',
+        log = models.ActivityLog(user_id=user.User_ID, entered=user.User_ID, operation='Check In' if user_id == '-404' else 'Check Out',
                                  status='Success')
 
         if user_id == '-404':
             user.Checked_In = True
             user.Last_In = right_now
         else:
             user.Checked_In = False
-            user.Total_Hours = ExpressionWrapper(F('Total_Hours') + (right_now - user.Last_In),
+            try:
```

Comment:
The way I read it, if this try block catches something it could potentially log someone out without updating the hours. Is that a risk worth taking? I think a 500 would be better

Reply:
discussed during lunch